WebAPI/Controllers/DTOs/EditAgendaItemDTO.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace WebAPI.Controllers.DTOs
+{
+    public class EditAgendaItemDTO
```

Comment:
name AgendaPointEditDTO is used in storage.

Reply:
renamed